WooCommerce/Classes/POS/Presentation/PointOfSaleCollectCashView.swift

```diff
@@ -120,6 +106,55 @@ struct PointOfSaleCollectCashView: View {
 }
 
 private extension PointOfSaleCollectCashView {
+    @ViewBuilder
+    var navigationHeader: some View {
+        if dynamicTypeSize.isAccessibilitySize {
```

Comment:
I tried to collapse this one into smaller subviews but ended causing more problems than anything else as they ended being too simple views, or need to add many conditionals so the design would stick in non-accessibility mode, so I just left the duplication for simplicity.

Reply:
Oh, that's nice! I've duplicated it for VStack and works great https://github.com/woocommerce/woocommerce-ios/pull/14935/commits/6f9f3fa9830de6d2c3f79892f9cc6ad4b642072c